### PR TITLE
Add Skip List for Aten Ops that are fused in nvFuser.

### PR DESCRIPTION
**Pull request by kevinstephano**
This Skip List (tuple) is added under the nvprims context manager.

### CODE REVIEW

File: torch/_prims/context.py [hunk 1]
```diff
@@ -175,15 +175,20 @@ def __torch_function__(
         # implementations.
         # There're other ways to implement this functionality,
         # see https://github.com/pytorch/pytorch/pull/82657#discussion_r939776417
+        decomp = False
```
Comment (jjsjann123) [inline torch/_prims/context.py:178]: nitpick: debug code should be removed.

**Review by jjsjann123 — APPROVED** [review]
LGTM. Minor comments on removing debug code.

File: torch/_prims/context.py (hunk 1 above)
Comment (kevinstephano) [reply torch/_prims/context.py:178]: Thanks for the catch. I forgot to remove that line.

**Review by mruberry — APPROVED** [review]
Stamped!

File: torch/_prims/context.py [hunk 2]
```diff
@@ -248,10 +248,21 @@ def _is_func_unsupported_nvfuser(
...
     def __init__(self, *, skip_ops=()):
+        aten_ops_to_skip = (
+            "aten.transpose.int",
+            "aten.t.default",
+            "aten.view.default",
+            "aten.unsqueeze.default",
+            "aten.permute.default",
+            "aten._log_softmax.default",
+            "aten._log_softmax_backward_data.default",
+            "aten.expand.default",
```
Comment (jjsjann123) [inline torch/_prims/context.py:259]: Can we add `aten.col2im` in the list because of #86332

Comment (jjsjann123) [reply torch/_prims/context.py:259]: This actually doesn't help here... it gets decomposed during fake tensor tracing and we hit this prior to our decomposition. :cry:

**Pull request closed without merging**